src/resources/views/columns/radio.blade.php

```diff
@@ -4,4 +4,4 @@
 	$displayValue = isset($column['options'][$entryValue]) ? $column['options'][$entryValue] : '';
 @endphp
 
-<td>{{ $displayValue }}</td>
+<span>{{ $displayValue }}</span>
```

Comment:
Why does this span not have NLs but the ones in the others do?

Reply:
Done. Your code styling OCD is clearly superior to mine :-)